src/main/resources/nl/dtls/fairdatapoint/database/rdf/repository/catalog/getDatasetThemesForCatalog.sparql

```diff
@@ -0,0 +1,6 @@
+prefix dcat: <http://www.w3.org/ns/dcat#>
+
+SELECT ?theme WHERE {
```

Comment:
@rajaram5 would it make sense to have a `DISTINCT(?theme)` here? The code probably stores the themes in a `Set`, so we wouldn't need to worry about duplication. Potential performance gain if there are **a lot** of themes?

Reply:
It would be better I guess. 

But still, you have to deal with duplications when you merge themes with theme taxonomies.

Reply:
Indeed, it is better to have `DISTINCT`

Reply:
I added DISTINCT